tests/common/test_event.py

```diff
@@ -412,19 +411,17 @@ def test_collect_events_should_ignore_invalid_event_files(self):
                 all(TestEvent._get_event_message(evt) == "A test telemetry message." for evt in event_list),
                 "The valid events were not found")
 
-            invalid_events = []
-            total_dropped_count = 0
-            for args, kwargs in mock_add_event.call_args_list: # pylint: disable=unused-variable
-                match = re.search(r"DroppedEventsCount: (\d+)", kwargs['message'])
-                if match is not None:
-                    invalid_events.append(kwargs['op'])
-                    total_dropped_count += int(match.groups()[0])
+            invalid_events = {}
+            for args in mock_warn.call_args_list:
+                if re.search('Failed to process event file', args[0][0]) is not None:
+                    invalid_events[args[0][1]] = args[0][1]
 
-            self.assertEqual(3, total_dropped_count, "Total dropped events dont match")
-            self.assertIn(WALAEventOperation.CollectEventErrors, invalid_events,
-                          "{0} errors not reported".format(WALAEventOperation.CollectEventErrors))
-            self.assertIn(WALAEventOperation.CollectEventUnicodeErrors, invalid_events,
-                          "{0} errors not reported".format(WALAEventOperation.CollectEventUnicodeErrors))
+            def assert_invalid_file_was_reported(filename):
+                self.assertIn(filename, invalid_events, '{0} was not reported as an invalid event file'.format(filename))
+
+            assert_invalid_file_was_reported("custom_script_utf-16.tld")
```

Comment:
This is failing, saying this file is not invalid

Reply:
A quick question, is it failing after the revert or before the revert?

Reply:
After the revert, same for below